pkg/agent/client.go

```diff
@@ -249,11 +252,35 @@ func (a *Client) Connect() (int, error) {
 	}
 	a.conn = conn
 	a.stream = stream
+	a.serverChannel = make(chan *client.Packet, xfrChannelSize)
+	a.serverError = nil
 	a.serverID = serverID
+	go a.writeToKonnServer()
 	klog.V(2).InfoS("Connect to", "server", serverID)
 	return serverCount, nil
 }
 
+func (a *Client) writeToKonnServer() {
```

Comment:
I think it is already hard to understand the naming in the code (proxy, server, agent, client, frontend, backend, remote etc), that's why I think the `writeToKonnServer` function name is not the best - also so far this Konn name is not appearing in the code. That's why I would like to suggest some other name like `writeToProxyServer` or something like that. What do you think? Thanks! Adam

Reply:
Done